framework/db/QueryBuilder.php

```diff
@@ -9,6 +9,7 @@
 
 use yii\base\InvalidParamException;
 use yii\base\NotSupportedException;
+use yii\db\Expression;
```

Comment:
Why?
`yii\db\QueryBuilder` and `yii\db\Expression` are under the same namespace.